src/components/ToolMenu/Draw/index.tsx

```diff
@@ -152,186 +159,207 @@ export const Draw: React.FC<DrawProps> = ({
     return <></>;
   }
 
+  const onToggleChange = (childProps: any) => {
+    setSelectedButton(childProps.name);
+  };
+
+  const attributeDrawer = (active: any) => {
```

Comment:
Can we type `active` to `boolean`?

```suggestion
  const attributeDrawer = (active: boolean) => {
```

Reply:
This should be fixed now